controllers/host/host_controller.go

```diff
@@ -1387,7 +1387,7 @@ func (r *HostReconciler) statusUpdateRequired(instance *starlingxv1.Host, host *
 			logHost.V(2).Info("host inSync, set unlock required")
 			status.StrategyRequired = cloudManager.StrategyUnlockRequired
 			strategyUpdated = true
-		} else if status.StrategyRequired != cloudManager.StrategyNotRequired {
+		} else if status.StrategyRequired == cloudManager.StrategyUnlockRequired && host.IsUnlockedEnabled() {
```

Comment:
Same comment here:
what's the expectation:
1 if strategy lock required
2 what if host in other status, failed/disabled etc.
When adding condition, please consider other conditions that cause code escape

Reply:
Reverted this condition with same improvement mentioned above.